test/benchmarks/driver_bench/src/runner.mts

```diff
@@ -14,8 +14,10 @@ type BenchmarkModule = {
   run: () => Promise<void>;
   afterEach?: () => Promise<void>;
   after?: () => Promise<void>;
+  tags?: string[];
 };
 
+
 const benchmarkName = snakeToCamel(path.basename(benchmarkFile, '.mjs'));
 const benchmark: BenchmarkModule = await import(`./${benchmarkFile}`);
 
```

Comment:
```js
const tags = benchmark.tags;
if (tags && !Array.isArray(tags) && tag.length > 0 && !tags.every(t => typeof t === 'string')) throw new Error();
```

mistakes happen while developing a benchmark and it is a nice thing to validate early so you don't learn of incorrect things after the benchmarks have run or when we do perf.send. Additionally, I figure we should enforce that if a tag array is given it has at least one item.

Reply:
Also, I don't have a strong reason for this but should we assert here that the tags are in sorted order, so that we always have them consistently ordered across benchmarks.

Reply:
I don't think the tags being sorted is a problem either way. Looking at the evergreen patch results, the table doesn't show the tags, but it does correctly apply the Contexts, but I will add the validation.